src/teams/teams.jl

```diff
@@ -0,0 +1,18 @@
+type Team <: GitHubType
+    name::Nullable{GitHubString}
+    description::Nullable{GitHubString}
+    privacy::Nullable{GitHubString}
+    permission::Nullable{GitHubString}
+    slug::Nullable{GitHubString}
+    id::Nullable{Int}
+end
+
+namefield(t::Team) = t.name
```

Comment:
`namefield` should return the field that's interpolated into the endpoint (for most API calls, if not all of them), so this should presumably be `t.id` since that's what you're using below.